addons/website_mass_mailing/tests/test_snippets.py

```diff
@@ -29,5 +29,6 @@ def test_snippet_newsletter_block_witih_edit(self):
         self.start_tour(
             self.env['website'].get_client_action_url('/'),
             "snippet_newsletter_block_with_edit",
-            login='admin'
+            login='admin',
+            step_delay=100,
```

Comment:
Any reason to add it globally on the test and not on the problematic step?

Which step if failing? The one after the call to `clickOnEditAndWaitEditMode`?

Then shouldn't the extra delay be added on this util? If it fails for this tour because entering edit mode is too slow, other tests could also fail the same way?

But actually, isn't `step_delay` always waiting the provided amount of time between each steps :grimacing: ? That would really be something to avoid if that's the case, at least add it only on the problematic step only.

So I guess the click arrive too soon after having entered edit mode?
Seems weird, edit mode JS should have been ready before clicking on it.

Seems like this is hiding the issue rather than really fixing it (it will break again once the js takes more than 100ms)
Why not but at least only on the problematic steps, not on every steps

Reply:
Yeah you're right, I'm currently trying something else.
I think I can fix this by adding an extra trigger.
Thanks for the feedback